edit/edit.js

```diff
@@ -1334,7 +1364,20 @@ function setStyleMeta(style) {
   $('#url').href = style.url || '';
 }
 
-function initWithStyle({style, codeIsUpdated}) {
+function initWithStyle({style}) {
+  // FIXME: what does codeIsUpdated do?
```

Comment:
codeIsUpdated is set in saveStyle on trivial operations like toggling a style, could be also due to an update that just moved some sections around without changing them.